Cargo.toml

```diff
@@ -43,7 +43,7 @@ tracing-limit = { path = "lib/tracing-limit" }
 # Tokio / Futures
 futures01 = { package = "futures", version = "0.1.25" }
 futures = { version = "0.3", default-features = false, features = ["compat", "io-compat"] }
-tokio01 = { package = "tokio", version = "0.1.22", features = ["io", "uds", "tcp", "rt-full", "experimental-tracing"], default-features = false }
+tokio01 = { package = "tokio", version = "0.1.22", features = ["io", "uds", "tcp", "rt-full", "experimental-tracing", "codec"], default-features = false }
```

Comment:
Why did we need to add `codec` here?

Reply:
We do use codec feature, [the test](https://github.com/timberio/vector/runs/793364942) before exposed that. I suspect somewhere in `trust-dns` dependencies chains we were depending on `tokio v0.1` with `codec` feature enabled, so cargo merged feature set to use only one version of `tokio`. Although this just my speculation as I've never observed this behavior before.